lib/file-store.js

```diff
@@ -74,10 +105,9 @@ module.exports.delete = async (file_id, authn_user_id) => {
  * @param {number} file_id - The file to get.
  * @return {object} An object with a buffer (of the file contents) and a file object.
  */
-module.exports.get = async (file_id) => {
-    debug(`get(): file_id=${file_id}`);
-
-    const params = { file_id };
+module.exports.get = async (file_id, assessment_instance_id, display_filename) => {
+    debug(`get(): file_id=${file_id} storage_type=${assessment_instance_id} display_filename=${display_filename}`);
```

Comment:
The only problem with the approach above is that we get a list of Assessments to pull IDs from res.locals, so I am not too sure if it is safe to pick the first item in the list. That would allow us to change query approaches to pull the right file data.

![image](https://user-images.githubusercontent.com/7872295/92017668-f7bb7400-ed08-11ea-8036-f5c8e019efcd.png)